vizro-core/src/vizro/models/types.py

```diff
@@ -43,6 +43,11 @@ def __init__(self, function, /, *args, **kwargs):
         """
         self.__function = function
         self.__bound_arguments = inspect.signature(function).bind_partial(*args, **kwargs)
+        # Below is required as otherwise kwargs provided to captured functions will sit under the key "kwargs"
+        kwargs_to_unfold = self.__bound_arguments.arguments.get("kwargs")
+        if kwargs_to_unfold:
+            self.__bound_arguments.arguments.update(kwargs_to_unfold)
+            del self.__bound_arguments.arguments["kwargs"]
```

Comment:
Will the changes from https://github.com/mckinsey/vizro/pull/121 fix this?

Reply:
Yes, we should be able to revert this change here 🙂